django_extensions/management/commands/show_urls.py

```diff
@@ -87,12 +99,20 @@ def handle(self, *args, **options):
             translation.activate(language)
             self.LANGUAGES = [(code, name) for code, name in getattr(settings, 'LANGUAGES', []) if code == language]
         else:
-            self.LANGUAGES = getattr(settings, 'LANGUAGES', ((None, None), ))
+            self.LANGUAGES = getattr(settings, 'LANGUAGES', ((None, None),))
 
         decorator = options['decorator']
         if not decorator:
             decorator = ['login_required']
 
+        if options['app_label'] and options['app_label'] not in settings.INSTALLED_APPS:
```

Comment:
Think we should use Django's app system for this ?
If people put `AppConfigs` in their `INSTALLED_APPS` this will not work.

https://docs.djangoproject.com/en/2.2/ref/applications/
https://docs.djangoproject.com/en/2.2/ref/applications/#django.apps.apps.get_app_config